discovery/gossiper.go

```diff
@@ -1277,6 +1277,11 @@ func (d *AuthenticatedGossiper) processChanPolicyUpdate(
 		)
 		edge.TimeLockDelta = uint16(policyUpdate.newSchema.TimeLockDelta)
 
+		// MaxHTLC is optional in channel policy update requests.
+		if policyUpdate.newSchema.MaxHTLC != 0 {
+			edge.MaxHTLC = policyUpdate.newSchema.MaxHTLC
```

Comment:
Also here, shouldn't we be careful not to interpret `0` as "leave unchanged"?

Reply:
If it's `0` here, it'll get set a few lines later in [`updateChannel`](https://github.com/lightningnetwork/lnd/blob/master/discovery/gossiper.go#L2429).